packages/lesswrong/components/questions/AnswersSorting.tsx

```diff
@@ -6,23 +6,15 @@ import * as _ from 'underscore';
 import type { Option } from '../common/InlineSelect';
 import { getCommentViewOptions } from '../../lib/commentViewOptions';
 import { useNavigate } from '../../lib/reactRouterWrapper';
-
```

Comment:
The changes in this file basically revert the changes to this file in https://github.com/ForumMagnum/ForumMagnum/pull/7908 (while keeping the copy updates)

I think the change was an attempt to unify sorting between comments and answers, but comments seem to use different views to do the sorting while answers use different `sortBy` values passed into a single view.